scripts/tauri-prebuild.mjs

```
@@ -128,9 +131,10 @@ try {
         outcome: cacheOutcome,
         source: cacheSource,
         buildDurationMs: Date.now() - prebuildStartedAt,
-      });
+      }, { projectRoot: root });
       console.log(`[release-cache] receipt ${finalized.receiptPath}`);
     } catch (error) {
+      if (isReleaseBuildCacheSafetyError(error)) throw error;
```

Biome (2.5.7)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)